meilisearch-lib/src/tasks/task_store/mod.rs

```diff
@@ -231,6 +233,19 @@ impl TaskStore {
         })
         .await?
     }
+
+    pub async fn dump(&self, dir_path: &Path) -> Result<()> {
+        let update_dir = dir_path.join("updates");
+        tokio::fs::create_dir(&update_dir).await?;
+        let updates_file = update_dir.join("data.jsonl");
+        dbg!(&updates_file);
+        let mut updates_file = tokio::fs::File::create(updates_file).await?;
+        for task in self.list_tasks(None, None, None).await? {
+            updates_file.write_all(&serde_json::to_vec(&task)?).await?;
+        }
```

Comment:
can we make this function synchronous, and write to the file using `serde_json::to_writer`?
also I am pretty sure you need a line break between each json

Reply:
an also use BufferedIo